lib/service_provider.rb

```diff
@@ -1,9 +1,6 @@
 require 'fingerprinter'
 
 class ServiceProvider
-  # currently acceptable encryption values are 'none' and 'aes256-cbc'
-  DEFAULT_ENCRYPTION = 'none'.freeze
```

Comment:
You could achieve the same effect of this entire PR by changing the default value here to `aes256-cbc`